pkg/sources/filesystem/filesystem.go

```diff
@@ -231,7 +235,7 @@ func (s *Source) scanFile(ctx context.Context, path string, chunksChan chan *sou
 // filepath or a directory.
 func (s *Source) Enumerate(ctx context.Context, reporter sources.UnitReporter) error {
 	for _, path := range s.paths {
-		fileInfo, err := os.Stat(filepath.Clean(path))
+		fileInfo, err := os.Lstat(filepath.Clean(path))
```

Comment:
question: Did you just update these for consistency?

Reply:
Yes